packages/runtime-html/src/observation/style-attribute-accessor.ts

```diff
@@ -53,6 +53,67 @@ export class StyleAttributeAccessor implements IAccessor<unknown> {
     }
   }
 
+  private spltStyleString(currentValue: string): [string, string][] {
+    const returnVal: [string, string][] = [];
+    const rx = /\s*([\w\-]+)\s*:\s*((?:(?:[\w\-]+\(\s*(?:"(?:\\"|[^"])*"|'(?:\\'|[^'])*'|[\w\-]+\(\s*(?:[^"](?:\\"|[^"])*"|'(?:\\'|[^'])*'|[^\)]*)\),?|[^\)]*)\),?|"(?:\\"|[^"])*"|'(?:\\'|[^'])*'|[^;]*),?\s*)+);?/g;
+    let pair: RegExpExecArray;
+    let style;
+    while ((pair = rx.exec(currentValue)!) != null) {
```

Comment:
`exec` never returns `undefined`, so this can be strict

```suggestion
    while ((pair = rx.exec(currentValue)!) !== null) {
```